common/assets/scss/overrides/_font-corrections.scss

```diff
@@ -8,3 +8,12 @@
   padding-top: 4px;
   padding-bottom: 4px;
 }
+
+.moj-organisation-nav {
```

Comment:
You can drop this surrounding definition. One of the ideas behind [BEM](http://getbem.com/) is that you don't need to nest things so heavily so it makes it easier to read.

Reply:
Also, noticed they've added some annoying margins onto the element which clash with the margins in our main container. 

This should fix the erroneous way spacing is being done:

```scss
.moj-organisation-nav {
  margin: 0;
  padding: 10px 0;
}
.moj-organisation-nav__link {
  padding-top: 2px;
}
```

I might try and do a PR to that repo to fix the spacing too.

Reply:
👍